testing/features/automated_release_process.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Automated Release Process
+
+This feature is to automate the design system release process, removing repeatable manual developer actions. 
+
+As a design system developer 
+I need to be able to specify that a design system release is Alpha or Public
+So that users understand that the design system is ready for either the public or develop environments
+ 
+As a design system developer 
+I need to have the option to release design system updates manually
+So that I am not reliant on 3rd party services to release updates automatically e.g. Github / Jenkins
+ 
+As a design system developer 
+I need to be able to stop the automated design system release process
+So that so that I can prevent errors and make manual changes after the process has started
```

Comment:
I'm unclear on the primary benefit of being able to stop the automated process, if this is a requirement. Have put some options here, any thoughts?

Reply:
I guess we would want to do this if we have made some sort of mistake that hasn't been captured before the release starts, eg a wrong version number or mis-labelling as `pre-release` instead of `stable release` perhaps?

So maybe change `prevent errors and make manual changes` to `prevent a bad release from being published` - @davidrapson  what do you think?

Reply:
Updated to "prevent a bad release from being published"